projects/components/src/sequence/sequence.ts

```diff
@@ -6,6 +7,18 @@ export interface SequenceSegment {
   start: number;
   end: number;
   color: string;
+  markers: Marker[];
+}
+
+export interface Marker {
+  id: string;
+  markerTime: number;
+  timestamps: string[];
+}
+
+export interface MarkerDatum {
+  marker: Marker;
+  origin: ElementRef;
```

Comment:
If we could use the approach we discussed earlier then that would be great. Mainly, a way to avoid passing an elementRef

Reply:
Will take a look in this.